xen/drivers/passthrough/iommu.c

```diff
@@ -129,7 +129,7 @@ static void __init parse_iommu_param(char *s)
     } while ( ss );
 }
 
-int iommu_domain_init(struct domain *d)
+int iommu_domain_init(struct domain *d, bool_t use_iommu)
```

Comment:
with [iommu: Call iommu construct() if use_iommu flag is set](https://github.com/xen-troops/xen/pull/48/commits/8bf040bd5d28fc2d7417609187f9ad79e4b601a1)